src/cargo/core/compiler/custom_build.rs

```diff
@@ -1,3 +1,36 @@
+//! A module about how to execute a custom build script and parse its output.
```

Comment:
Side note: If `custom_build.rs` is really about build scripts, should we make that more obvious, whether thats renaming this to `build_script.rs` or `build_rs.rs`?

Reply:
Had the same thought. I could see custom build/build script/build.rs co-exist everywhere. `build_script.rs` gets my vote and I can send a PR fixing all occurrences then.